google/cloud/logging_v2/client.py

```diff
@@ -347,18 +350,22 @@ def get_default_handler(self, **kw):
         """
         monitored_resource = kw.pop("resource", detect_resource(self.project))
 
-        if (
-            isinstance(monitored_resource, Resource)
-            and monitored_resource.type == _GAE_RESOURCE_TYPE
-        ):
-            return AppEngineHandler(self, **kw)
-        elif (
-            isinstance(monitored_resource, Resource)
-            and monitored_resource.type == _GKE_RESOURCE_TYPE
-        ):
-            return ContainerEngineHandler(**kw)
-        else:
-            return CloudLoggingHandler(self, resource=monitored_resource, **kw)
+        if isinstance(monitored_resource, Resource):
+            if monitored_resource.type == _GAE_RESOURCE_TYPE:
+                return AppEngineHandler(self, **kw)
+            elif monitored_resource.type == _GKE_RESOURCE_TYPE:
+                return ContainerEngineHandler(**kw)
+            elif (
+                monitored_resource.type == _GCF_RESOURCE_TYPE
+                and sys.version_info[0] == 3
+                and sys.version_info[1] >= 8
+            ):
+                # Cloud Functions with runtimes > 3.8 supports structured logs on standard out
+                # 3.7 should use the standard CloudLoggingHandler
```

Comment:
probably worth mentioning what standard CloudLoggingHandler does compared to StructuredLogHandler.

Reply:
good point, added